src/App.js

```diff
@@ -1,10 +1,33 @@
+
+import logo from './logo.svg';
 import './App.css';
-import Header from './Header';
+import Dashboard from "./Pages/Dashboard/Dashboard.js";
 
 function App() {
   return (
-    <div className="App">
-      <Header />
+    <div className="app">
+      <header className="app-header">
+        <img src={logo} className="app-logo" alt="logo" />
```

Comment:
As we start to build the project, I kindly suggest you deleting this default part from React setup 😃